tests/basics/run_all.py

```diff
@@ -121,6 +121,9 @@
             search_mode=search_mode,
             needs_2to3=needs_2to3,
         )
+        
+        if search_mode.abortIfExecuted():
+            break
```

Comment:
I hate how this has to modify every usage, but I accept it, because it's my fault, since the skipping dialog is part of using the search mode too, right now. One day, using the search mode, should be more common code, and this only be in one or two places max.